src/main/java/WayofTime/alchemicalWizardry/AlchemicalWizardry.java

```diff
@@ -499,10 +499,13 @@ public class AlchemicalWizardry {
     public static BlockStack[] specialAltarBlock = new BlockStack[7];
 
     public static int lpPerSelfSacrifice = 200;
-    public static int lpPerSelfSacrificeSoulFray = 20;
     public static int lpPerSelfSacrificeFeatheredKnife = 100;
     public static int lpPerSacrificeBase = 500;
     public static int lpPerSacrificeWellOfSuffering = 10;
+    public static int VillagerMultiplierSacrificeWellOfSuffering = 12;
```

Comment:
This is weirdly subjective and I don't personally like it, why are villagers just randomly so significantly better than monsters? You don't exactly give reasoning